static/temp/foods.json

```diff
@@ -0,0 +1,259 @@
+[
+  {
+    "name": "Banana ripe, slightly ripe, raw",
+    "category": {
+      "name": "Fruit and Fruit Juices",
+      "icon": "EggAltOutlinedIcon"
+    },
+    "overview": {
+      "coolFood": {
+        "icon": "IconName",
+        "text": "This is a cool food",
+        "moreInfo": {
+          "description": "Cool Food Meals are a simple and delicious way to help the planet.",
+          "linkText": "Learn More",
+          "link": "https://coolfood.org/consumer/"
+        }
+      },
+      "nutrientScore": "B",
+      "climateImpact": {
+        "title": "Climate Impact",
+        "emissions": "200kg CO2",
+        "emissionsUnit": "per serving",
+        "comparisons": [
+          {
+            "title": "0.2 gallons of diesel consumed",
+            "icon": "OilBarrelOutlinedIcon",
+            "order": 0
+          },
+          {
+            "title": "0.2 gallons of diesel consumed",
+            "icon": "DirectionsCarFilledOutlinedIcon",
+            "order": 1
+          }
+        ]
+      },
+      "tips": [
+        {
+          "name": "0.2 gallons of diesel consumed",
+          "icon": "OilBarrelOutlinedIcon",
+          "description": [
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            },
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            },
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            }
+          ]
+        },
+        {
+          "name": "0.2 gallons of diesel consumed",
+          "icon": "OilBarrelOutlinedIcon",
+          "description": [
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            },
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            },
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            }
+          ]
+        }
+      ]
+    },
+    "images": ["/img1", "/img2", "/img3"],
+    "nutrition": {
+      "nutrientName": "Banana ripe, slightly ripe, raw",
```

Comment:
This is not `nutrientName`. This is food name which is already represented as `name` in the JSON above. Shall we delete it?

Reply:
done

Reply:
done